bindings/python/rgbmatrix/core.cpp

```diff
@@ -1,4 +1,4 @@
-/* Generated by Cython 0.29.6 */
+/* Generated by Cython 0.29.2 */
```

Comment:
If possible, I'd try to keep the same cython version as before (0.29.6), so that the differences are only reflecting the changes due to the new property.

Reply:
Rebuilt using the v0.29.6 of cython.